lis/surfacemodels/land/ac.7.2/ac72_readcrd.F90

```diff
@@ -0,0 +1,264 @@
+    enddo
+
+    ! PathNameSimul
+    call ESMF_ConfigFindLabel(LIS_config, "AquaCrop.7.2 input path:", rc = rc)
+    do n=1, LIS_rc%nnest
+        call ESMF_ConfigGetAttribute(LIS_config, &
+            AC72_struc(n)%PathNameSimul, rc=rc)
+        call LIS_verify(rc, "AquaCrop.7.2 input path: not defined")
+    enddo
+
+    ! CO2_Filename
+    call ESMF_ConfigFindLabel(LIS_config, "AquaCrop.7.2 CO2_Filename:", rc = rc)
+    do n=1, LIS_rc%nnest
+        call ESMF_ConfigGetAttribute(LIS_config, &
+            AC72_struc(n)%CO2_Filename, rc=rc)
+        call LIS_verify(rc, "AquaCrop.7.2 CO2_Filename: not defined")
+    enddo
+ 
+    ! Management_Filename
+    call ESMF_ConfigFindLabel(LIS_config, "AquaCrop.7.2 Management_Filename:", rc = rc)
+    do n=1, LIS_rc%nnest
+        if (rc == 0) then
+            call ESMF_ConfigGetAttribute(LIS_config, &
+                AC72_struc(n)%Management_Filename, rc=rc)
+             ! change lis none to AquaCrop (None)
+             if ((AC72_struc(n)%Management_Filename .eq. 'none') .or. &
+                 (AC72_struc(n)%Management_Filename .eq. 'None')) then
+                 AC72_struc(n)%Management_Filename = '(None)'
+             endif 
+        else
+            write(LIS_logunit, *)'[INFO] AC72 Management_Filename: not defined, default management'
+            AC72_struc(n)%Management_Filename = '(None)'
+        endif
+    enddo
+ 
+    ! Irrigation_Filename
+    call ESMF_ConfigFindLabel(LIS_config, "AquaCrop.7.2 Irrigation_Filename:", rc = rc)
+    do n=1, LIS_rc%nnest
+        if (rc == 0) then
+            call ESMF_ConfigGetAttribute(LIS_config, &
+                 AC72_struc(n)%Irrigation_Filename, rc=rc)
+             ! change lis none to AquaCrop (None)
+             if ((AC72_struc(n)%Irrigation_Filename .eq. 'none') .or. &
+                 (AC72_struc(n)%Irrigation_Filename .eq. 'None')) then
+                 AC72_struc(n)%Irrigation_Filename = '(None)'
+             endif 
+        else
+            write(LIS_logunit, *)'[INFO] AC72 Irrigation_Filename: not defined, no irrigation'
+            AC72_struc(n)%Irrigation_Filename = '(None)'
+        endif
+    enddo
+ 
+    ! AquaCrop model soil parameter table
+    call ESMF_ConfigFindLabel(LIS_config, "AquaCrop.7.2 soil parameter table:", rc = rc)
+    do n=1, LIS_rc%nnest
+        call ESMF_ConfigGetAttribute(LIS_config, AC72_struc(n)%soil_tbl_name, rc=rc)
+        call LIS_verify(rc, "AquaCrop.7.2 soil parameter table: not defined")
+    enddo
+
+ 
+    ! soil classification scheme
+    do n=1, LIS_rc%nnest
+        ios = nf90_get_att(nids(n), NF90_GLOBAL, 'SOILTEXT_SCHEME', soil_scheme_name)
+        call LIS_verify(ios, 'Error in nf90_get_att: SOILTEXT_SCHEME')
+        if (trim(soil_scheme_name) .eq. "STATSGO") then
+          AC72_struc(n)%soil_scheme_name = "STAS"
+        else
+          write(LIS_logunit, *) "Fatal error: currently, only STATSGO soil scheme is supported by AquaCrop!"
```

Comment:
Again, just curiosity: is this choice just heritage from Noah, used to make AquaCrop work?

Reply:
Yes, this is heritage from our usage of Noah-MP I think, right @mbechtold?

Reply:
Right.